config/prod/tom-cvat-ubuntu.yaml

```diff
@@ -0,0 +1,53 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v2.j2
+stack_name: tom-cvat-ubuntu
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "{{stack_group_config.default_key_pair}}"
+  AMIId: "{{stack_group_config.default_ubuntu_ami}}"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "p3.2xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "500"
+  # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
+  # AMIId: "ami-0565af6e282977273"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  # VpcSubnet: "PublicSubnet"
+  # (Optional) Set true to enable daily backups (default: false)
+  # BackupEc2: "true"
+  # (Optional) Number of daily backups to keep (default: 7) (valid range: 1-180)
+  # SnapshotCount: "14"
+  Ec2InstanceId: "i-03510ca53eb38da83"
```

Comment:
Are these the right configurations?

Reply:
no, you don't want these.  you'll need to setup the load balancer with a separate, follow on PR.